package.json

```diff
@@ -46,7 +46,7 @@
     "build": "shx rm -rf ./distribution && tsc -p tsconfig.production.json && madge ./distribution --circular",
     "build:fast": "tsc -p tsconfig.production.json",
     "build:flow-types":
-      "cp source/danger.d.ts source/_danger.d.ts; sed -ie 's/api: GitHub/api: any/g' source/_danger.d.ts; npx flowgen source/_danger.d.ts -o distribution/danger.js.flow; node scripts/update_flow_types.js",
+      "cp source/danger.d.ts source/_danger.d.ts && sed -ie 's/api: GitHub/api: any/g' source/_danger.d.ts && npx -p github:orta/flowgen#union_Strings flowgen source/_danger.d.ts -o distribution/danger.js.flow && node scripts/update_flow_types.js",
```

Comment:
Cant' figure out how to make npx work with my forked version of flowgen, `npx  flowgen@orta/flowgen#union_Strings source/_danger.d.ts -o distribution/danger.js.flow` was my last attempt